shard-operator/src/main/java/com/redhat/service/smartevents/shard/operator/BridgeExecutorServiceImpl.java

```diff
@@ -87,11 +89,17 @@ public void createBridgeExecutor(ProcessorDTO processorDTO) {
             // create or update the secrets for the bridgeExecutor
             createOrUpdateBridgeExecutorSecret(bridgeExecutor, processorDTO);
         } else {
-            LOGGER.info("BridgeExecutor '{}' already exists. Notifying manager that it is ready.", processorDTO.getId());
-            processorDTO.setStatus(ManagedResourceStatus.READY);
-            managerClient.notifyProcessorStatusChange(processorDTO).subscribe().with(
-                    success -> LOGGER.debug("Ready notification for BridgeExecutor '{}' has been sent to the manager successfully", processorDTO.getId()),
-                    failure -> LOGGER.error("Failed to send updated status to Manager for entity of type '{}'", ProcessorDTO.class.getSimpleName(), failure));
+            Boolean isBridgeExecutorReady = existing.getStatus()
```

Comment:
I believe `existing.getStatus().isReady()` does this already.

I was doing much the same here too... but your PR has beaten me to it; thank-you.